sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1755,15 +1755,23 @@ object functions {
   def cbrt(columnName: String): Column = cbrt(Column(columnName))
 
   /**
-   * Computes the ceiling of the given value.
+   * Computes the ceiling of the given value of `e` to `scale` decimal places.
+   *
+   * @group math_funcs
+   * @since 3.3.0
+   */
+  def ceil(e: Column, scale: Column): Column = withExpr { Ceil(e.expr, scale.expr) }
```

Comment:
We can use `UnresolvedFunction`

Reply:
Could you please give an example for the `UnresolvedFunction`, please ?

Reply:
`def ceil(e: Column, scale: Column): Column = withExpr { UnresolvedFunction(Seq("ceil"), Seq(e.expr, scale.expr), isDistinct = false) }`